sql/2021/caching/cache_control_directives_invalid.sql

```diff
@@ -0,0 +1,33 @@
+#standardSQL
+# List of invalid Cache-Control directive names.
+SELECT
+  client,
+  total_requests,
+  total_using_cache_control,
+  directive_name,
+  directive_occurrences,
+  directive_occurrences / total_using_cache_control AS pct_of_cache_control,
+  directive_occurrences / total_requests AS pct_of_total_requests
+FROM (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    directive_name,
+    COUNT(0) AS directive_occurrences,
+    SUM(COUNT(0)) OVER (PARTITION BY _TABLE_SUFFIX) AS total_requests,
+    SUM(COUNTIF(TRIM(resp_cache_control) != '')) OVER (PARTITION BY _TABLE_SUFFIX) AS total_using_cache_control
```

Comment:
These are not `total_requests` as per recent discussions, as you have an `UNNEST` clause.

Running your query shows 1,030,983,798 requests for desktop, but we have 584,173,241 actual requests. Hence showing how this is wrong IMHO.

Can you double check this?